reports/cfgmgmnt.go

```diff
@@ -0,0 +1,71 @@
+package reports
+
+import (
+	"encoding/json"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// Facts stores output of puppet & chef facts command
+type Facts map[string]interface{}
+
+// GetFacts will exec into the Facts interface
+func GetFacts(confTool, cmdPath, cmdArgs string) (Facts, error) {
+	args := strings.Split(cmdArgs, " ")
+	cmd := exec.Command(cmdPath, args...)
+
+	o, err := cmd.Output()
+	if err != nil {
+		return Facts{}, errors.Wrap(err, "exec puppet facts")
+	}
+
+	var facts Facts
```

Comment:
this could just be `var facts map[string]interface{}` Is there a reason for `type Facts`?

Reply:
Copy/paste from previous implementation. I can clean that up on my next pass since we now know how all the tools are going to work.